package.json

```diff
@@ -13,7 +13,8 @@
     "start": "node src/index.js",
     "db:setup": "node src/scripts/createCollections.js",
     "db:delete": "node src/scripts/deleteCollections.js",
-    "db:reset": "npm run db:delete && npm run db:setup"
+    "db:reset": "npm run db:delete && npm run db:setup",
+    "parse-references": "node src/scripts/parseModelInfo.js"
```

Comment:
Would suggest expanding this name later if we also have it parse data to help with search functionality. This can be done later when that work is necessary.